### PR TITLE
Adds must_select_whole_seat_block to AvailabilityMeta

### PR DESCRIPTION
**Pull request by stevgouws**
Does what it says on the tin: adds `must_select_whole_seat_block` to the AvailabilityMeta class.

### CODE REVIEW

**Review by cakemanny — APPROVED** [review]
Can't fault this :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [2.8.5] - 2018-05-29
```
Comment (cakemanny) [inline CHANGELOG.md:9]: Actually this!

**Review by cakemanny — CHANGES_REQUESTED** [review]

**Review by cakemanny — CHANGES_REQUESTED** [review]
Actually I didn't review this very well at all

You also want to add it to the `__init__` method and the the class documentation

also there is a test in `tests/test_client.py` as well

```
~/src/pyticketswitch (master) % ag contiguous_seat_selection_only
tests/test_availability.py
107:        "contiguous_seat_selection_only": True,
129:        assert meta.contiguous_seat_selection_only is True

tests/test_client.py
1045:            'contiguous_seat_selection_only': True,
1064:        assert meta.contiguous_seat_selection_only is True

docs/users/advanced.rst
380:    >>> meta.contiguous_seat_selection_only
386:When :class:`contiguous_seat_selection_only
387:<pyticketswitch.availability.AvailabilityMeta.contiguous_seat_selection_only>`

pyticketswitch/availability.py
11:        contiguous_seat_selection_only (bool): indicates that the backend
36:    def __init__(self, contiguous_seat_selection_only=True, currency=None,
40:        self.contiguous_seat_selection_only = contiguous_seat_selection_only
68:        inst.contiguous_seat_selection_only = data.get('contiguous_seat_selection_only', True)
```
can leave the advanced.rst for later I think

**Comment by cakemanny** [comment]
and don't worry about the ci build error for now. looks like that's never been set up
can come back to that

File: CHANGELOG.md (hunk 1 above)
Comment (stevgouws) [reply CHANGELOG.md:9]: 🤦 jeez, pre-lunch brain. Thanks!

**Comment by stevgouws** [comment]
> Actually I didn't review this very well at all
> 
> You also want to add it to the `__init__` method and the the class documentation
> 
> also there is a test in `tests/test_client.py` as well
> 
> ```
> ~/src/pyticketswitch (master) % ag contiguous_seat_selection_only
> tests/test_availability.py
> 107:        "contiguous_seat_selection_only": True,
> 129:        assert meta.contiguous_seat_selection_only is True
> 
> tests/test_client.py
> 1045:            'contiguous_seat_selection_only': True,
> 1064:        assert meta.contiguous_seat_selection_only is True
> 
> docs/users/advanced.rst
> 380:    >>> meta.contiguous_seat_selection_only
> 386:When :class:`contiguous_seat_selection_only
> 387:<pyticketswitch.availability.AvailabilityMeta.contiguous_seat_selection_only>`
> 
> pyticketswitch/availability.py
> 11:        contiguous_seat_selection_only (bool): indicates that the backend
> 36:    def __init__(self, contiguous_seat_selection_only=True, currency=None,
> 40:        self.contiguous_seat_selection_only = contiguous_seat_selection_only
> 68:        inst.contiguous_seat_selection_only = data.get('contiguous_seat_selection_only', True)
> ```
> 
> can leave the advanced.rst for later I think

Updated 👉  https://github.com/ingresso-group/pyticketswitch/commit/a50a64fb3b90d5b2598daec1284a334031674ae9

**Review by cakemanny — APPROVED** [review]

**Comment by cakemanny** [comment]
I pulled Steven's branch and ran the tests and made sure they passed
(in case anyone is reading this)

**Pull request merged**